### PR TITLE
Fix project parsing to handle missing groupid

### PR DESCRIPTION
**Pull request by weshaggard**

### CODE REVIEW

**Review by chidozieononiwu — APPROVED** [review]

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -14,7 +14,12 @@ function Get-java-PackageInfoFromRepo ($pkgPath, $serviceDirectory)
     $projectData.load($projectPath)
     $projectPkgName = $projectData.project.artifactId
     $pkgVersion = $projectData.project.version
-    $pkgGroup = $projectData.project.groupId
+    if ($projectData.project.psobject.properties.name -contains "groupId") {
```
Comment (chidozieononiwu) [inline eng/scripts/Language-Settings.ps1:17]:
```suggestion
    if ($projectData.project -contains "groupId") {
```

Comment (weshaggard) [reply eng/scripts/Language-Settings.ps1:17]: That code doesn't work. the contains operator needs to be on a list of the property names.

**Review by chidozieononiwu — APPROVED** [review]

File: eng/scripts/Language-Settings.ps1 (hunk 1 above)
Comment (chidozieononiwu) [reply eng/scripts/Language-Settings.ps1:17]: You are right. I was a little confused by the introduction of `psobject`. But the code is fine the way it was.

**Comment by weshaggard** [comment]
build issues are because of https://github.com/Azure/azure-sdk-for-java/pull/20065.

**Pull request merged**